### DISCUSSION TITLE
Unpacked Size 5MByte?

### DISCUSSION THREAD

**Discussion by guettli** [orchidjs/tom-select#129] [General]
According to this page, the unpacked size is 5 MByte:

https://www.npmjs.com/package/tom-select

This looks way too much.

For example autocomplete.js is only 780k: https://www.npmjs.com/package/autocomplete.js

The JS files via CDN of both projects have roughly the same size.

**Comment by oyejorge** [comment]
Notice also that on that page `Total Files = 267` for Tom Select and `Total Files = 72` for autocompete.js

When you install Tom Select via npm (eg `npm install tom-select`) you install all scss/plugins/esm/cjs/etc. When you use Tom Select in a project though, you'll only use one or two of those files.

**Reply by oyejorge** [reply]
One more note about the sizes: [autocomplete.min.js](https://cdn.jsdelivr.net/autocomplete.js/0/autocomplete.min.js) is about 57 kb and [tom-select.complete.min.js](https://cdn.jsdelivr.net/npm/tom-select@1.7.5/dist/js/tom-select.complete.min.js) is about 44 kb
